protocol-impl/src/main/java/io/zeebe/protocol/impl/record/value/job/JobRecord.java

```diff
@@ -39,6 +39,7 @@
   private final StringProperty workerProp = new StringProperty("worker", EMPTY_STRING);
   private final LongProperty deadlineProp = new LongProperty("deadline", -1);
   private final IntegerProperty retriesProp = new IntegerProperty(RETRIES, -1);
+  private final LongProperty retryBackoffProp = new LongProperty("retryBackoff", 0);
```

Comment:
The other properties use `-1` as the default value. Why do you suggest `0` here instead?

Reply:
Good point! I'll set it as `-1`

Reply:
Done